app/helpers/application_helper.rb

```diff
@@ -248,6 +248,8 @@ def holding_block_search(document)
         info << content_tag(:div, search_location_display(holding, document), class: 'library-location', data: { location: true, record_id: document['id'], holding_id: id })
       end
       block << content_tag(:li, info.html_safe, data: { availability_record: check_availability, record_id: document['id'], holding_id: id, aeon: aeon_location?(location) })
+      cdl_placeholder = content_tag(:span, '', class: 'badge badge-primary', 'data-availability-cdl' => true)
+      block << content_tag(:li, cdl_placeholder.html_safe)
```

Comment:
@hectorcorrea do you think we should add here the use_alma flag `if Rails.configuration.use_alma`? If we don't add it then we have in production an unused cdl element `<span class="badge badge-primary" data-availability-cdl="true"></span>`. It doesn't break anything but it is an unused span. What do you think?

Reply:
yes, good idea